index.html

```diff
@@ -80,7 +80,156 @@ <h1 class="mn__title">
             /></a>
           </li>
         </ul>
+        <div class="arrow_down">
+          <img
+            src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"
+            alt="circled arrow-down icon"
+          />
+        </div>
+      </section>
+      <section class="works__section">
+        <div class="recent__works">
+          <h2>My Recent Works</h2>
+        </div>
+        <div class="bar__under"></div>
+        <div class="proj__container" id="proj__1">
+          <img class="proj__img" src="img/Work 1.png" alt="1st project image" />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
+            <a class="see__proj__btn btns" href="#"><span>See Project</span></a>
+          </div>
+        </div>
+        <div class="proj__container" id="proj__2">
+          <img
+            class="proj__img"
+            src="img/Work 2.webp"
+            alt="2nd project image"
+          />
+          <div class="proj__list">
+            <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+            <div class="proj__lang">
+              <a class="bt" href="#">Ruby on rails</a>
+              <a class="bt" href="#">CSS</a>
+              <a class="bt" href="#">JavaScript</a>
+              <a class="bt" href="#">HTML</a>
+            </div>
```

Comment:
[OPTIONAL] I like your design but I think the best way to implement this block is to use a combination  ul tag and li tag.The ul tag  is used for lists.
What do you think about this :arrow_down: 
```
<ul class="proj__lang">
              <li class="bt">Ruby on rails</li>
              <li class="bt">CSS</li>
              <li class="bt">JavaScript</li>
              <li class="bt">HTML</li>
  </ul>
```